docs/databricks.md

```diff
@@ -0,0 +1,120 @@
+======================================================
+This is a getting started guide to XGBoost4J-Spark on Databricks. At the end of this guide, the reader will be able to run a sample Apache Spark application that runs on NVIDIA GPUs on Databricks.
+
+Prerequisites
+-------------
+* Apache Spark 2.4+ running in DataBricks Runtime 5.3 or 5.4 ML with GPU hardware. Make sure it matches requirements below. Use nodes with 1 GPU each.
+* Hardware Requirements
+  * NVIDIA Pascal™ GPU architecture or better
+  * Multi-node clusters with homogenous GPU configuration
+* Software Requirements
+  * Ubuntu 16.04/CentOS
+  * NVIDIA driver 410.48+
+  * CUDA V10.0/9.2
+  * NCCL 2.4.7
+
+* The number of GPUs per node dictates the number of Spark executors that can run in that node. Each executor should only be allowed to run 1 task at any given time. 
+
+Get Application Jar and Dataset
+-------------------------------
+* Jars: Please download the Rapids XGBoost4j Spark jars.  There are 3 jars.  
+Download the required jars into a local directory. Databricks ML Runtime supports cuda 9.2 so download the correct jars. If the below commands don't work then you can go to Maven Central and search for the 3 jars: ai.rapids.cudf version 0.8-Beta ,ai.rapids.xgboost4j - version 0.90-Beta, and ai.rapids.xgboost4j-spark 0.90-Beta.  
+
+```
+$ wget https://search.maven.org/remotecontent?filepath=ai/rapids/cudf/0.8-Beta/cudf-0.8-Beta.jar
+$ wget https://search.maven.org/remotecontent?filepath=ai/rapids/xgboost4j/0.90-Beta/xgboost4j-0.90-Beta.jar
+$ wget https://search.maven.org/remotecontent?filepath=ai/rapids/xgboost4j-spark/0.90-Beta/xgboost4j-spark-0.90-Beta.jar
+``` 
+
+* Download Dataset: https://rapidsai-data.s3.us-east-2.amazonaws.com/spark/mortgage.zip
+
+```
+$ wget  https://rapidsai-data.s3.us-east-2.amazonaws.com/spark/mortgage.zip
+$ unzip mortgage.zip
+$ find . -type f -print|sort
+./data/mortgage/csv/test/mortgage_eval_merged.csv
+./data/mortgage/csv/train/mortgage_train_merged.csv
+``` 
+
+* Copy the data into Databricks DBFS:
+
+```
+Go to "Import and Explore Data" 
+Create New Table - default location is /FileStore/tables
+Select mortgage_eval_merged.csv and mortgage_train_merged.csv to upload.
+```
+
+* Upload the XGBoost-4j Spark jars
+
+```
+Go to "Import Library"
+Select "Upload" and then "Jar"
+Select the 3 XGBoost Spark jars and upload them. Save the file names and locations for the step below.
+```
+
+* Add a Startup init script  
+In a terminal create an init script that has commands to copy the jars you imported above over the existing Databricks provided XGBoost jars. Note you will have to change the jar names below to the ones you uploaded.  If you didn't save the names go to the Databricks file browser and look in /FileStore/jars/ If you are using a runtime other then Databricks 5.3 or 5.4 you will have to check the versions of the databricks provided jars in /databricks/jars and update the script accordingly.  
```

Comment:
I think there's a period missing between "/FileStore/jars/" and "If you are using a runtime..."